renderer/components/Login/Login.js

```diff
@@ -0,0 +1,66 @@
+import React, { useRef } from 'react'
+import PropTypes from 'prop-types'
+import { FormattedMessage } from 'react-intl'
+import { Flex } from 'rebass/styled-components'
+import { useFormError } from 'hooks'
+import { Button, CenteredContent, Message, Text } from 'components/UI'
+import { Form, PasswordInput } from 'components/Form'
+import ArrowRight from 'components/Icon/ArrowRight'
+import ZapLogo from 'components/Icon/ZapLogo'
+import messages from './messages'
+
+const Login = ({ login, loginError, clearLoginError, ...rest }) => {
+  const formApiRef = useRef(null)
+
+  useFormError(loginError, clearLoginError, formApiRef)
+
+  const handleSubmit = ({ password }) => login(password)
+
+  return (
+    <Form getApi={api => (formApiRef.current = api)} onSubmit={handleSubmit} width={1}>
+      {({ formState: { submits, error } }) => {
+        const willValidateInline = submits > 0
+        return (
+          <CenteredContent {...rest} mx="auto" width={11 / 16}>
+            <Flex alignItems="center" flexDirection="column" px={0} width={5 / 8} {...rest}>
```

Comment:
`...rest` is being applied twice - once to the outer `CenteredContent` and then again to this `Flex`. It should probably only be applied on the outer most element.